h/services/annotation_moderation.py

```diff
@@ -2,13 +2,29 @@
 
 from __future__ import unicode_literals
 
+from sqlalchemy import exists
+
 from h import models
 
 
 class AnnotationModerationService(object):
     def __init__(self, session):
         self.session = session
 
+    def hidden(self, annotation_id):
+        """
+        Check if an annotation id is hidden.
+
+        :param annotation_id: The id of the annotation to check.
+        :type annotation: unicode
+
+        :returns: true/false boolean
+        :rtype: bool
+        """
+        query = self.session.query(exists().where(
+            models.AnnotationModeration.annotation_id == annotation_id))
```

Comment:
This clearly works, but TBH I find this form of the exists query quite hard to read. You can also use `Query.exists()`, which would save you an import and work like this:

```python
q = self.session.query(models.AnnotationModeration).filter_by(annotation_id=annotation_id)
return self.session.query(q.exists()).scalar()
```

I'm certainly not going to fuss over this one. I just personally find this form a little easier to read.